main.rb

```diff
@@ -0,0 +1,8 @@
+require './app'
+
+class Main
+  def main
+    app = App.new
+    app.library_menu
+  end
+end
```

Comment:
- [ ] I'm sorry that the previous reviewer didn't point this out, but your app doesn't work as expected. I'm marking this as required because is important that this functionality gets implemented in order to continue with the next milestones:

Currently, when a resource (book, person) is created in your app, the app exits. However, according to the [video](https://www.youtube.com/watch?v=vkkgrhD6aXQ), Your app should only exit when the user chooses the Exit option, Otherwise, the menu should keep printing itself after the user interacts with any option.

This behavior is important because at the moment there's no way of preserving the data, so I'm unable to create more than one resource e.g. a Book and a Person to finally create a new Rental. 😱 

You can achieve this by using a loop that breaks when the user press 7. Please consider applying this! 🤓 🔧